ReactiveCocoa/RACObjCRuntimeUtilities.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+typedef void (^rac_receiver_t)(void);
```

Comment:
This is deliberately made so that it can be extended as something like `(RACUnsafeArgumentTuple) -> Void`.